azure/firewall-policy/variables.tf

```diff
@@ -44,6 +44,7 @@ variable "dns" {
     servers       = optional(list(string))
     proxy_enabled = optional(bool)
```

Comment:
Heavily suggest that we default this to "on" unless we have a reason not to, servers can be optional as they'll inherit from the VNet.

Reply:
@tb-ans What is the justification for this as this appears to go against the default?

Reply:
Also, it's worth noting that in its current form (tyla's change not implemented), the block where this variable is used (line 14 on `main.tf`) should be a dynamic block. If Tyla's change is implemented then it makes sense to keep that block as is.

Reply:
Discussed - Makes more sense to set to true as default